devops/jobs/CreatePingdomAlerts.groovy

```diff
@@ -0,0 +1,97 @@
+/*
+
+    Variables without defaults are marked (required)
+
+    Variables consumed for this job:
+        * NOTIFY_ON_FAILURE: (Required - email address)
+        * SECURE_GIT_CREDENTIALS: (Required - jenkins name of git credentials)
+        * PINGDOM_EMAIL: (Required - The Pingdom Email)
+        * PINGDOM_API_KEY: (Required - Pingdom API Key)
+        * PINGDOM_PASSWORD: (Required - Pingdom Password)
+        * PINGDOM_ALERT_CONFIG_FILE (The pingdom Alert config file)
```

Comment:
I assumed this would be a config file in the configuration-internal repo
Given that, we need to clone that repo too, right?